### ISSUE TITLE
Unknown layer type: Warping

### ISSUE THREAD

**Issue by kspeng** [BVLC/caffe#6731]
### Issue summary
I got the latest version caffe and set WITH_PYTHON_LAYER := 1 in makefile.config. However, I still get the following error:

------
F0325 16:49:50.756420  6840 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Warping (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Clip, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, Swish, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***
    @     0x7fb0d2d7f5cd  google::LogMessage::Fail()
    @     0x7fb0d2d81433  google::LogMessage::SendToLog()
    @     0x7fb0d2d7f15b  google::LogMessage::Flush()
    @     0x7fb0d2d81e1e  google::LogMessageFatal::~LogMessageFatal()
    @     0x7fb0d33cf46c  caffe::Net<>::Init()
    @     0x7fb0d33d0b7e  caffe::Net<>::Net()
    @     0x7fb0d352f22a  caffe::Solver<>::InitTrainNet()
    @     0x7fb0d35306f5  caffe::Solver<>::Init()
    @     0x7fb0d3530a0f  caffe::Solver<>::Solver()
    @     0x7fb0d33a1b21  caffe::Creator_AdamSolver<>()
    @           0x40a778  train()
    @           0x407568  main
    @     0x7fb0d1522830  __libc_start_main
    @           0x407e39  _start
    @              (nil)  (unknown)
train_resnet_50by2_pool_tvl1.sh: line 8:  6840 Aborted                 (core dumped) $TOOLS/caffe train --solver=solver_resnet50by2_pooladam_tvl1.prototxt --gpu=0,1,2,3 --log_dir=logs/
------
Is there anything I missed? Thanks.

**Comment by cqray1990** [comment]
> ### Issue summary
> I got the latest version caffe and set WITH_PYTHON_LAYER := 1 in makefile.config. However, I still get the following error:
> 
> ## F0325 16:49:50.756420 6840 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Warping (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Clip, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, Swish, TanH, Threshold, Tile, WindowData)
> *** Check failure stack trace: ***
> @ 0x7fb0d2d7f5cd google::LogMessage::Fail()
> @ 0x7fb0d2d81433 google::LogMessage::SendToLog()
> @ 0x7fb0d2d7f15b google::LogMessage::Flush()
> @ 0x7fb0d2d81e1e google::LogMessageFatal::~LogMessageFatal()
> @ 0x7fb0d33cf46c caffe::Net<>::Init()
> @ 0x7fb0d33d0b7e caffe::Net<>::Net()
> @ 0x7fb0d352f22a caffe::Solver<>::InitTrainNet()
> @ 0x7fb0d35306f5 caffe::Solver<>::Init()
> @ 0x7fb0d3530a0f caffe::Solver<>::Solver()
> @ 0x7fb0d33a1b21 caffe::Creator_AdamSolver<>()
> @ 0x40a778 train()
> @ 0x407568 main
> @ 0x7fb0d1522830 __libc_start_main
> @ 0x407e39 _start
> @ (nil) (unknown)
> train_resnet_50by2_pool_tvl1.sh: line 8: 6840 Aborted (core dumped) $TOOLS/caffe train --solver=solver_resnet50by2_pooladam_tvl1.prototxt --gpu=0,1,2,3 --log_dir=logs/
> Is there anything I missed? Thanks.

i get the same errorss,and i also change WITH_PYTHON_LAYER,but it failed ,did you solver it ?thank you so much

**Comment by cqray1990** [comment]
@kspeng did you solve it ?can you help me answer this question?thank you  so much

**Comment by kspeng** [comment]
@cqray1990 Sorry, I gave it up and used other way.
